app-backend/database/src/main/scala/com/azavea/rf/database/tables/Uploads.scala

```diff
@@ -47,6 +42,97 @@ class Uploads(_tableTag: Tag) extends Table[Upload](_tableTag, "uploads")
   val visibility: Rep[Visibility] = column[Visibility]("visibility")
```

Comment:
Yeah so the way it works is `metadata` is now free-form key value pairs of any sort, and I was right about the `circe` magic -- it's kind of cool I think